genesyscloud/routing_utilization/resource_routing_utilization_utils.go

```diff
@@ -182,18 +122,3 @@ func getSdkUtilizationTypes() []string {
 	sort.Strings(types)
 	return types
 }
-
-// TODO: remove when routing skill group is refactored
-func buildHeaderParams(routingAPI *platformclientv2.RoutingApi) map[string]string {
```

Comment:
Not used anymore, pretty sure the mention to routing skill group was just copy and paste.

The actual routing skill group is using BuildHeaderParams (uppercase B), defined in routing_skill_group.go.

So this guy here can be removed.